ngrok/src/config/http.rs

```diff
@@ -420,6 +424,12 @@ impl HttpTunnelBuilder {
         self
     }
 
+    /// Set the policy for this edge.
+    pub fn policy(&mut self, policy: impl Borrow<Policy>) -> &mut Self {
+        self.options.common_opts.policy = Some(policy.borrow().to_owned());
+        self
+    }
```

Comment:
It feels weird to borrow this only to clone it with `to_owned()`. I think we should just take a plain `Policy` and let the user clone it if they need to reuse it. That's more "The Rust Way".

Reply:
Ah, I grabbed that from the oauth/oidc pattern: https://github.com/ngrok/ngrok-rust/commit/b9e90a853d138fcc75eeb4ac8f6df31e7cf3bb71#diff-efbdb6c60163d74bd9e27471661bd42e341e3be73bcb917a4373d1d3b5aeba27R278

Reply:
Changed all the `Borrow` to `Into`/`TryInto`.

Reply:
Oh, gross, did I do that? :sweat_smile: 

Hindsight is 20/20 I guess